README.md

```diff
@@ -1,12 +1,116 @@
 # mlflow
 
+**MLflow** is an open source platform for managing the end-to-end machine learning lifecycle.
+
 ## Description
 
-TODO: Describe your charm in a few paragraphs of Markdown
+This charm provides **MLflow Tracking Server** and **Model Registry**, that can be described as
+these functions:
+ * Tracking experiments to record and compare parameters along with the model results.
+ * Providing a central model store to collaboratively manage the full lifecycle of an MLflow Model, 
+   including model versioning, stage transitions, and annotations.
+   
+For more information visit [MLflow Documentation][MLflow-docs]
 
 ## Usage
 
-TODO: Provide high-level usage, such as required config or relations
+Right now there no published version in [charmhub] and it needs to be
+manually build with `charmcraft`.
+
+    charmcraft build -v
```

Comment:
This command returned an error because of dep conflicts (already discussed in another channel and requirements.txt is about to be updated).